docs/learning/frontend/QuantityFormatting.md

```diff
@@ -21,7 +21,31 @@ There are nine built-in quantity types (see [QuantityType]($frontend)). The Quan
 
  Custom quantity types that implement the [CustomQuantityTypeDefinition]($frontend) interface may also be registered with the QuantityFormatter, see method `registerQuantityType`. See example implementation of a custom type [here](https://github.com/iTwin/itwinjs-core/blob/5905cb1a48c4d790b5389d7f0ea141bc3ce95f23/test-apps/ui-test-app/src/frontend/api/BearingQuantityType.ts).
 
-#### Overriding Default Formats
+#### Migrating from QuantityType to KindOfQuantity
+
+Starting in iTwin.js 5.0, we encourage developers to move away from using `QuantityType`, and to instead use `KindOfQuantity` [EC full names](https://www.itwinjs.org/bis/ec/ec-name/#full-name).
+
+Here is a table of replacements for each `QuantityType`:
```

Comment:
Should we also highlight that the new way provides access to much more formats and not just the ones listed in the table? That would also explain why the deprecation of `QuantityType` is being done in the first place.

Reply:
I added a paragraph explaining why we are moving away from `QuantityType`, it mentions being able to access more formats and make it easier